### ISSUE TITLE
Accessibility issues with our error summary 

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1853]
### Description of the bug

Discovered by SSB during UU testing 🥇 
- [x] List is coded as an ordered list, but visually it is an unordered list. Should be changed to `<ul>`
- [x] The list is nested inside a `<p>` element. This is not allowed. 
- [x] `Role="alert"` is placed on the `<li>` element. We should use `aria-live="assertive"` on a text element
Text suggestion: "Det har oppstått feil i skjemaet, du må rette de her feilene før du kan gå videre"
- [ ] Consider swapping with the design system error summary. (Are any of these issues present there? @Febakke ) 

### Steps To Reproduce

Go to a app with validation
Trigger validation 
inspect the alert

**Comment by bjosttveit** [comment]
These problems do not seem to exist in the error summary, I think these were fixed there a long time ago. However, they do appear to be present in the error messages directly below components, so fixing them there instead

**Comment by olavflar** [comment]
Is this in prod?
